site/test-suite-report.html

```diff
@@ -0,0 +1,238 @@
+<!doctype html>
+<html>
+  <head>
+    <style>
+      td {
+        text-align: center;
+      }
+    </style>
+  </head>
+  <body>
+    <input type="checkbox" id="core-only"></input><label for="core-only" title="Skip tests that require type-specific implementation">core only</label>
+
+    <script crossorigin="anonymous" integrity="sha384-VRhlkYEqoAXsWzoORIQL0gHdX0BvQoQG89GxEj7I4xlffttgMvoMsQr7E/IWGK9T" src="https://unpkg.com/chroma-js@2.4.2/chroma.min.js"></script>
+    <script crossorigin="anonymous" integrity="sha384-I7E8VVD/ismYTF4hNIPjVp/Zjvgyol6VFvRkX/vR+Vc4jQkC+hVqc2pM8ODewa9r" src="https://unpkg.com/@popperjs/core@2.11.8/dist/umd/popper.min.js"></script>
+    <script crossorigin="anonymous" integrity="sha384-AiTRpehQ7zqeua0Ypfa6Q4ki/ddhczZxrKtiQbTQUlJIhBkTeyoZP9/W/5ulFt29" src="https://unpkg.com/tippy.js@6.3.7/dist/tippy-bundle.umd.min.js"></script>
+    <script type="module">
```

Comment:
This repository is a bit of a hodgepodge anyway so I'm not sure quality matters much. But it would probably be cleaner to just ship this separately as a static JS file.